_partials/_psql-installation-linux.md

```diff
@@ -0,0 +1,74 @@
+### Install psql on Linux
```

Comment:
How many times is this partial used, Rajie? I ony see it once, but perhaps there are future use cases on other pages that I'm unaware of.

If we don't use a partial at least twice, I would prefer to inline it, because each partial has a slight negative impact on our site performance

Reply:
@charislam I plan to use the partial here https://docs.timescale.com/timescaledb/latest/how-to-guides/connecting/psql/. Did not want to add too many files to this PR. I will create a new PR later.